### ISSUE TITLE
Custom Blender Dataset Failing to Train

### ISSUE THREAD

**Issue by chengine** [ashawkey/torch-ngp#90]
Hello. Thanks for the great work. I've generated a custom dataset through Blender, which to me looks to be in the proper form. Using the original instant-ngp repo, I am able to train it just fine. However, using this repo, I am getting massive losses (as well as a warning that lr_scheduler.step() is called before optimizer.step(), something that does not occur when training the ficus from the original NeRF dataset). I am able to train the original NeRF datasets just fine, just not my custom dataset. The losses for my data are also printed for every step within the epoch, as opposed to just one loss per epoch that is updating in place. A snip of my dataset is attached. Any help is appreciated. Thanks.
[val.zip](https://github.com/ashawkey/torch-ngp/files/9227791/val.zip)

**Comment by ashawkey** [comment]
@chengine Hi, can you post the full command you use to train? also, can you try to uncomment this line https://github.com/ashawkey/torch-ngp/blob/main/nerf/provider.py#L241 and post the visualization of poses?

**Comment by chengine** [comment]
Hi! Thanks for the response. My command is `python main_nerf.py data/nerf_synthetic/smurf --workspace smurf_nerf -O --bound 1.5 --scale 0.8 --dt_gamma 0`. Attached is the pose viz. Thanks for your help.
![image](https://user-images.githubusercontent.com/73979029/182049983-13da6d74-f993-4447-a4d1-a3b5246df091.png)

**Comment by chengine** [comment]
Followup: I ran the exact same data generation script on another mesh. torch-ngp trains fine on that data, so it seems to be dataset specific. I visualized both sets of images through opencv and they seem to be fine. The only difference I can see is that size of the images in the dataset of interest are very large (on the order of MB) compared to images from other datasets.

**Comment by chengine** [comment]
I managed to fix the issue, so I will close this thread.

**Comment by ashawkey** [comment]
good to hear that!

**Comment by zhihaolee** [comment]
> I managed to fix the issue, so I will close this thread.

Hi [chengine](https://github.com/chengine), could you share the solution? We are having bad results on custom datasets.
